helix-rest/src/main/java/org/apache/helix/rest/client/CustomRestClientFactory.java

```diff
@@ -32,26 +37,32 @@
  */
 public class CustomRestClientFactory {
   private static final Logger LOG = LoggerFactory.getLogger(CustomRestClientFactory.class);
-
   private static CustomRestClient INSTANCE = null;
 
-  private CustomRestClientFactory() {
+  protected CustomRestClientFactory() {
   }
 
   public static CustomRestClient get() {
     if (INSTANCE == null) {
       synchronized (CustomRestClientFactory.class) {
         if (INSTANCE == null) {
           try {
+            // Here int has been used for timeout value because setConnectTimeout,
+            // setConnectionRequestTimeout and setSocketTimeout are getting int as input
+            final int httpRequestTimeout =
+                HelixUtil.getSystemPropertyAsInt(RestSystemPropertyKeys.HTTP_TIMEOUT_MS, 60 * 1000);
```

Comment:
Let's put this number to some constant file or at least to be a variable

Reply:
Sure. Moved to constant file.